ansible/deploy_todoApp.yaml

```diff
@@ -0,0 +1,62 @@
+---
+- name: Deploy To Do app
+  hosts: webserver
+  remote_user: ec2-user
+  vars_files:
+    - ./trello_secrets.yml
+
+  tasks: 
+    - name: Install Git
+      yum:
+        name: git
+        state: present
+      become: true
+    
+    - name: Install Python 3
+      yum:
+        name: python3
+        state: present
+      become: true
+
+    - name: Install Poetry
+      ansible.builtin.shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: /home/ec2-user/.local/bin/poetry
+
+    - name: Create directory for the To-Do App code
+      ansible.builtin.file:
+        path: /opt/todoapp
+        state: directory
+        mode: '0755'
+        owner: ec2-user
+      become: true
+
+    - name: Get the latest version of your code with Git
+      ansible.builtin.git:
+        repo: 'https://github.com/CreaTiVeUK/DevOps-Course-Starter.git'
+        dest: /opt/todoapp
+        version: module-3
```

Comment:
Longer term you'd probably want this with another branch - but it'll suffice here!

Reply:
Do I just remove "version: module-3" from here, to always pick up the latest version?